interfaces/HO-Portal/src/app/models/q-and-a.models.ts

```diff
@@ -0,0 +1,33 @@
+export enum AnswerType {
+  // Translate the types used in the API to internal, proper types:
+  Number = 'numeric',
+  Text = 'text',
+  Date = 'date',
+  Enum = 'dropdown',
+  phoneNumber = 'tel',
+  email = 'email',
+}
+
+export class Question {
+  code: string;
+  answerType: AnswerType;
+  label: string;
+  placeholder?: string;
+  pattern?: string;
+  options: QuestionOption[] | null;
+}
+
+export class QuestionOption {
+  value: string;
+  label: string;
+}
+
+export class Answer {
+  code: string;
+  value: string;
+  label: string;
+}
+
+export class AnswerSet {
+  [code: string]: Answer;
+}
```

Comment:
If all these aren't used, I'd not include them.
As they will get 'out of date' and only add confusion later.